paddle/fluid/platform/device_context.h

```diff
@@ -506,6 +509,7 @@ class MKLDNNDeviceContext : public CPUDeviceContext {
   mkldnn::engine engine_;
   std::shared_ptr<BlobMap> p_blobmap_;
   std::shared_ptr<std::mutex> p_mutex_;
+  bool block_next_cache_clearing = false;
```

Comment:
Our private members are named with a trailing underscore, like line 509 - line 512.

Could you rename it to "block_next_cache_clearing_" ?